src/arrayMethodJoin.js

```diff
@@ -5,7 +5,25 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let result = '';
+
+    let separatorNew = separator;
+
+    if (separator === undefined) {
+      separatorNew = ',';
+    }
```

Comment:
this same result can be neatly achieved by declaring a default value for function parameter (line 7):

function(separator = ',')

Reply:
@AndrzejPec thank you! all changed 👍